tools-in-path/test.sh

```diff
@@ -7,7 +7,7 @@ if [ -f /etc/profile ]; then
 fi
 
 set -euo pipefail
-
+IFS=$'\n\t'
```

Comment:
Why did you add this?

Reply:
I assume because it's generally added next to `set -euo pipefail` as part of ["bash strict mode"](http://redsymbol.net/articles/unofficial-bash-strict-mode/)?

Reply:
Should we be adding it to all tests then?

Reply:
I would be okay with that. Doesn't seem very high priority though.